### DISCUSSION TITLE
Can bus

### DISCUSSION THREAD

**Discussion by extremesss10** [gusmanb/logicanalyzer#50] [Ideas]
Has there been any consideration to adding can bus analysis?

**Comment by gusmanb** [comment]
Hi.

I have no device with CAN nearby, but if you can provide me some captures I can add it, the protocol itself is not very complex to implement and the plugin system makes creating the protocol analyzers very easy.

Also, if you want to implement it by yourself the Wiki contains all the info on [how to do it](https://github.com/gusmanb/logicanalyzer/wiki/06---The-LogicAnalyzer-program#creating-custom-analyzers).

Cheers.

**Reply by cdwijs** [reply]
Sigrok has some captures of CAN data here:
http://sigrok.org/gitweb/?p=sigrok-dumps.git;a=tree;f=can;hb=e2c49eb88327ac4eaaa41c160b9e3faa1d9ca1da

For reference, their CAN protocol decoder is here:
http://sigrok.org/gitweb/?p=libsigrokdecode.git;a=tree;f=decoders/can;h=a9b52f173e2ca7df101087af1da8e8e5e117402f;hb=0235970293590f673a253950e6c61017cefa97df
